### ISSUE TITLE
Accessibility issue: Heading levels should only increase by one on /create-content/adding-news-pages/news-view-grid-view-filter

### ISSUE THREAD

**Issue by accesswatch** [accesswatch/qs-proposal#359]
## What
An accessibility scan flagged the element `<h4 class="card-title text-midnight h5 mb-0">
    <span class="field field--name-title field--type-string field--label-hidden">Researchers convene at U of A to tackle &apos;forever chemicals&apos; at National PFAS Conference</span>

  </h4>` because heading levels should only increase by one. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/heading-order?application=playwright.

[View screenshot](https://github.com/accesswatch/qs-proposal/blob/gh-cache/.screenshots/e7307d90-0b7e-4f38-9e2f-6aadb7720d2f.png)

To fix this, ensure the order of headings is semantically correct.

Specifically:

Fix any of the following:
-   Heading order invalid

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
